openapi.json

```diff
@@ -14,6 +14,60 @@
 	}],
 	"paths": {
 		"/load-test": {
+			"get": {
```

Comment:
Please also add an example of usage together with an explanation to the doc [docs/Kangal-user-flow.md](docs/Kangal-user-flow.md)